reflectable/lib/src/builder_implementation.dart

```diff
@@ -5430,8 +5430,19 @@ Future<ResolvedLibraryResult> _getResolvedLibrary(
   // mentioned in dart-lang/build#2634. If we do not fetch a fresh session
   // then the code generation stops with an `InconsistentAnalysisException`
   // if there is more than one entry point.
-  final freshLibrary =
-      await resolver.libraryFor(await resolver.assetIdForElement(library));
-  final freshSession = freshLibrary.session;
-  return await freshSession.getResolvedLibraryByElement(freshLibrary);
+  var attempts = 0;
```

Comment:
How sad that this is necessary.

I'm not sure I understand the underlying problem fully. But at least we should have a reference to where this trick was borrowed from:
```suggestion
  // This hack was suggested in https://github.com/google/built_value.dart/issues/941#issuecomment-731077220.
  var attempts = 0;
```

Reply:
Done.